### ISSUE TITLE
Self-hosted shared item management in organization

### ISSUE THREAD

**Issue by remyrd** [bitwarden/core#333]
Hi there, I started hosting a bitwarden server for my family with the 5 people organization subscription.
The server is running the latest 1.21
Web version is 2.0
Desktop is 1.4 (windows)
Android is 1.17.1

The following is currently happening:
- Me, the owner, can not retrieve my collections in the android nor windows native. I created my account first, then created an organization, but can only see shared collections/items in the browser, even after syncing explicitly from within the apps.
- None of us can delete items nor upload attachments in the organization. The error message is related to `insufficient rights`. This happens regardless of which collection the item is assigned to.
- Collections are not displayed when attempting to share individual items from my personal vault, which prevents me from sharing them. On the other hand, selecting multiple items with the option `share selected` puts them into the `unassigned` collection, and feature the same problems as above (unable to delete, attach files).

The mssql logs seem OK, no new logs in `errorlogs` are generated when these events occur, same for other non-binary logfiles.

I'm wondering if there's an easy way to connect to the database and fix the user roles manually..

In the past when I was the only user in the organization, I was able to create/delete a bunch of items, include attachments, etc. If my memory is good, that was just before the release of 1.21 core.
I will try downgrading the core to 1.20 and report back..

**Comment by kspearrin** [comment]
Are the user's assigned with access rights to these collections? From your description of the problem it sounds like no.

**Comment by remyrd** [comment]
I set myself as `owner` for all items. The rest are `owners` in collections they are interested in. None has the `read-only` field checked. Maybe there is another form for this I'm not aware of besides `Manage > People > ex@mp.le`

**Comment by kspearrin** [comment]
Can you reproduce the problem on vault.bitwarden.com ? I am not able to.

**Comment by remyrd** [comment]
I can't. vault.bitwarden.com works like a charm. It surprised me collections are displayed in the landing page, this is not the case in my instance. 
I think the whole organization is corrupted in my backend: none of my shared items are displayed in other clients, a new user with full access rights in the org. sees none of the collections or shared items, I can't attach files to my own shared items, etc.

Looks like the best thing to do right now is re-create the organization and re-import the items, see where did it all go wrong and report back.

**Comment by remyrd** [comment]
Hello again, 
So after updating to the newest web/core, I noticed a warning: `organization disabled`.
After looking around it turned out the licence had changed from the time the free trial ended and the money was actually debited from my account.
Once I updated the licence and executed `updatedb` with the `bitwarden.sh` provided script, the warning disappeared along with the issues concerning the organization.
Current versions are
- core: 1.22.0
- web: 2.1.1
